src/platform/user/profile/vet360/util/transactions.js

```diff
@@ -57,6 +57,11 @@ export const LOW_CONFIDENCE_ADDRESS_ERROR_CODES = new Set([
 
 export const DECEASED_ERROR_CODES = new Set(['MVI300']);
 
+export const INVALID_EMAIL_ADDRESS_ERROR_CODES = new Set([
+  'EMAIL304',
```

Comment:
I included the `EMAIL304` error code here since it's also an indicator of an invalid email address. This error code was also already used in `personal-information/PersonalInformation.edit-email-address.unit.spec.jsx `